### PR TITLE
Save files produced in scheduled build

### PR DESCRIPTION
**Pull request by davidwalter2**

### CODE REVIEW

**Comment by kdlong** [comment]
You cancelled it because you were still checking some things?

**Comment by kdlong** [comment]
One additional thing I would like to do is save the kerberos password in a keytab on ~/private and then use that to kinit from the CI. For some reason, though, I can't figure that out. I've don't it in the past and it worked, but I can't get it to work from lxplus8s10. Was following these instructions: https://kb.iu.edu/d/aumh

**Comment by davidwalter2** [comment]
> You cancelled it because you were still checking some things?

I think most of it will only run in the scheduled build and not in the PR build, so I don't know how this can be tested properly

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -642,9 +648,20 @@ jobs:
...
-      - name: clean
+      - name: copy clean plots
         run: |
           echo "xrdcp --parallel 4 -R -f $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR/* root://eosuser.cern.ch//$EOS_DIR/$LOCAL_WEB_DIR"
           xrdcp --parallel 4 -R -f $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR/* root://eosuser.cern.ch//$EOS_DIR/$LOCAL_WEB_DIR
+          echo "Removing temp directory $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR"
+          rm -r $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR
+          
+      - name: save analysis files
+        if: github.event_name != 'pull_request' && github.event.schedule == '0 3 * * 2,4,6'
```
Comment (kdlong) [inline .github/workflows/main.yml:659]: Does the first condition add anything?

**Comment by kdlong** [comment]
> > You cancelled it because you were still checking some things?
> 
> I think most of it will only run in the scheduled build and not in the PR build, so I don't know how this can be tested properly

I think we can just merge and see what happens for the scheduled builds in the future, it's not that big of a deal if it breaks

File: .github/workflows/main.yml (hunk 1 above)
Comment (davidwalter2) [reply .github/workflows/main.yml:659]: I'm not sure, I have seen it like this at another place and just copied the line

**Pull request merged**